src/ResourceManager/CognitiveServices/AzureRM.CognitiveServices.psd1

```diff
@@ -75,10 +75,11 @@ FunctionsToExport = @()
 CmdletsToExport = 'Get-AzureRmCognitiveServicesAccount', 
                'Get-AzureRmCognitiveServicesAccountKey', 
                'Get-AzureRmCognitiveServicesAccountSkus', 
+			   'Get-AzureRmCognitiveServicesAccountUsage', 
```

Comment:
Please also add your new cmdlet to the netcore psd1: https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/CognitiveServices/AzureRM.CognitiveServices.Netcore.psd1

Reply:
Added